src/focm.c

```diff
@@ -31,7 +31,7 @@
 #include "ui_utils.h"
 #include "bands.h"
 
-bool no_multi() {
+bool no_multi(spot *s) {
```

Comment:
`setcontest.c` has a `static bool no_multi(spot *data)`, let make it accessible and use here too. just to avoid duplication.

Reply:
Did you think something is like in 0ee226e6?

Reply:
exactly